StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1609PropertyDocumentationMustHaveValue.cs

```diff
@@ -20,6 +20,7 @@
     /// <para>A violation of this rule occurs when the <c>&lt;value&gt;</c> tag for a property is missing.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: I believe this one can be generated, at least as an element containing no text. The result would then produce warning SA1610, but at least the user wouldn't have to type the XML portion manually.

```csharp
/// <value></value>
```